src/main/java/org/roaringbitmap/Container.java

```diff
@@ -122,6 +122,16 @@ public int andCardinality(Container x) {
     }
   }
 
+  /**
+   * Returns the cardinality of the XOR between the passed container and this
+   * container without materialising a temporary container.
+   * @param other other container
+   * @return the cardinality of the symmetric difference of the two containers
+   */
+  public int xorCardinality(Container other) {
+    return getCardinality() + other.getCardinality() - 2 * andCardinality(other);

```

Comment:
My idea was to implement ```xorCardinality``` for each type of container vs another container... but on second thoughts, given that ```getCardinality```  is cheap... except for RunContainer... I think the improvement won't be very big... Let's keep it like this... one can try to implement and benchmark later...